Monika After Story/game/script-ch30.rpy

```diff
@@ -1391,7 +1401,7 @@ label ch30_reset:
             # NOTE: we may add additional costume logic in here if needed
             # TODO: this is bad for o31 rests actually
 
-            if not persistent._mas_force_clothes:
+            if not persistent._mas_force_clothes and not persistent._mas_o31_in_o31_mode and not mas_isO31():
```

Comment:
Yeah, so the TODO that's here is referring to the fact that we should make o31's autoload change clothes appropraitely depending on the costume. I want to avoid adding additional conditionals here cause that makes `ch30_reset` more confusing.